orttraining/orttraining/python/training/ortmodule/_graph_execution_manager.py

```diff
@@ -92,6 +92,10 @@ def __init__(self, module):
         # flag to enable symbolic shape inference for dynamic shape inputs to improve performance
         self._run_symbolic_shape_infer = True
 
+        # Use torch.onnx.OperatorExportTypes.ONNX_FALLTHROUGH to allow custom autograd.Functions'.
+        # Use torch.onnx.OperatorExportTypes.ONNX if pure ONNX is needed.
+        self._onnx_export_type = onnx_export_type
```

Comment:
This is a continuation of the ORTModule.__init__ comment above.
If we have a separate API to enable custom autograd, that setting could be done in that API too.

Reply:
[wechi]

Reply:
After the email discussion we had, this change will most likely to be reverted, right? we will keep ONNX as default

Reply:
Not really. Without ONNX_FALLTHROUGH, tests still fail and @BowenBao is looking into the reason. For now, we just assume we still need `onnx_export_type` in this PR. We can remove it in another PR.

Reply:
May we remove ```onnx_export_type``` as an argument to the constructor (until we have a more generic way of configuring the ```ORTModule```) and instead add an experimental method to control the ```onnx_export_type```? An existing example of the experimental method to control the log level:
https://github.com/pytorch/ort/blob/main/torch_ort/experimental/debug.py#L50

Reply:
Yep. We follow the suggested pattern and rename it to `enable_custom_autograd_function` (type: bool).